test/integration/scheduler_perf/README.md

```diff
@@ -68,6 +68,25 @@ To produce a cpu profile:
 make test-integration WHAT=./test/integration/scheduler_perf KUBE_TIMEOUT="-timeout=3600s" KUBE_TEST_VMODULE="''" KUBE_TEST_ARGS="-alsologtostderr=false -logtostderr=false -run=^$$ -benchtime=1ns -bench=BenchmarkPerfScheduling -cpuprofile ~/cpu-profile.out"
 ```
 
+### How to configure bechmark tests
+
+Configuration file located under config/performance-config.yaml contains a list of templates.
+Each template allows to set:
+- node manifest
+- manifests for an initial and testing pod
+- number of nodes, number of initial and testing pods
+- templates for PVs and PVCs
+- feature gates
+
+See `simpleTestCases` data type implementation for available configuration for each template.
+
+Initial pods (optionally with PVs) create a state of a cluster before the scheduler performance measurement can begin.
```

Comment:
I think the `optionally with PVs` can be removed. Different test cases can optionally have their specific configurations, and PV is only one of those cases.

Reply:
Removed